met-api/src/met_api/utils/roles.py

```diff
@@ -56,4 +56,6 @@ class Role(Enum):
     VIEW_ALL_ENGAGEMENTS = 'view_all_engagements'  # Allows user access to all engagements including draft
     SHOW_ALL_COMMENT_STATUS = 'show_all_comment_status'  # Allows user to see all comment status
     EXPORT_TO_CSV = 'export_to_csv'  # Allows users to export comments to csv
+    EXPORT_ALL_CAC_FORM_TO_SHEET = 'export_all_cac_form_to_sheet'  # Allows users to export CAC form to csv
+    EXPORT_CAC_FORM_TO_SHEET = 'export_cac_form_to_sheet'  # Allows users to export CAC form to csv
```

Comment:
whats the difference between this two?

Reply:
To do this:
![image](https://github.com/bcgov/met-public/assets/91914654/f341d63f-2c71-4fb9-ad42-8822c7f9dc44)